azure-pipelines.yml

```diff
@@ -42,280 +33,236 @@ resources:
+                HelixAccessToken: $(HelixApiAccessToken)
+            displayName: Validate Helix
+
+          - job: Validate_Signing
+            strategy:
+              matrix:
+                Test_Signing:
+                  _BuildConfig: Debug
+                  _SignType: test
+                Real_Signing:
+                  _BuildConfig: Release
+                  _SignType: real
+            steps:
+              - checkout: self
+                clean: true
+              - task: CopyFiles@2
+                displayName: Copy test packages to artifacts directory
+                inputs:
+                  sourceFolder: $(Build.SourcesDirectory)\src\validation\resources
+                  targetFolder: $(Build.SourcesDirectory)\artifacts\packages\$(_BuildConfig)\NonShipping
+              - powershell: eng\common\build.ps1
+                  -configuration $(_BuildConfig)
+                  -restore
+                  -prepareMachine
+                  -sign
+                  -ci
+                  /p:DotNetSignType=$(_SignType)
+                  /p:TeamName=DotNetCore
+                  /p:OfficialBuildId=$(BUILD.BUILDNUMBER)
+                displayName: Sign packages
+
+    - stage: Create_BAR_ID_Tag
+      displayName: Create BAR ID Tag
+      condition: succeededOrFailed()
       jobs:
-      - template: /eng/common/templates/job/job.yml
+      - template: /eng/common/templates-official/job/job.yml@self
         parameters:
-          name: Validate_Publishing
-          displayName: Validate Publishing
-          timeoutInMinutes: 240
-          pool:
-            name: $(PoolProvider)
-            demands: ImageOverride -equals windows.vs2019.amd64
+          name: Create_BAR_ID_Tag
+          displayName: Create BAR ID Tag
           variables:
             - group: Publish-Build-Assets
             - group: DotNetBot-GitHub
-            - name: BARBuildId
-              value: $[ dependencies.setupMaestroVars.outputs['setReleaseVars.BARBuildId'] ]
-            - name: skipComponentGovernanceDetection
-              value: true
           steps:
-            - template: /eng/common/templates/post-build/setup-maestro-vars.yml
             - checkout: self
               clean: true
-            - powershell: eng\validation\test-publishing.ps1
-                -buildId $(BARBuildId)
+            - powershell: eng/create-baridtag.ps1
                 -azdoToken $(dn-bot-dotnet-build-rw-code-rw)
-                -azdoUser "dotnet-bot"
-                -azdoOrg "dnceng"
-                -azdoProject "internal"
                 -barToken $(MaestroAccessToken)
                 -githubPAT $(BotAccount-dotnet-bot-repo-PAT)
-    - stage: Promote_Arcade_To_Latest
-      displayName: Promote Arcade
-      dependsOn:
+              displayName: Create BAR ID Tag
+              name: Create_BAR_ID_Tag
+    - template: /eng/common/templates-official/post-build/post-build.yml@self
```

Comment:
nit: aren't the `policheck` and `credscan` checks done here redundant when using the 1ES PT❔